### PR TITLE
Add optional tests

### LINKED ISSUE #107: Add benchmark-like tests for CMEPS-mediator/NEMS-mediator evaluation
The model components:
FV3_CCPP_C384
0.25-degree MOM6
0.25-degree CICE5
0.5-degree WW3

Model starting dates:
January 1, 2012, 2013
April 1, 2012, 2013
July 1, 2012, 2013
October 1, 2012, 2013

The following tests will be added:
1. Eight cold-start runs using NEMS mediator
2. Eight 35-day runs using NEMS mediator
3. Eight cold-start runs using CMEPS mediator
4. Eight 35-day runs using CMEPS mediator

The tests will be set up and run on Orion.

### PR DESCRIPTION
**Pull request by binli2337**
close #107

### CODE REVIEW

File: tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100 [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#
+#  CPLD_FV3_CCPP_384_MOM6_CICE_WW3_35D_BMARK_RT test
+#
+
+export TEST_DESCR="Coupled FV3-CCPP-MOM6-CICE-WW3 system"
+
+export CNTL_DIR="RT-Baselines_35d_bmwav_2012"
+export CNTLMED_DIR="RT-Baselines_35d_bmwav_2012/RESTART"
+
+export LIST_FILES=""
```
Comment (DeniseWorthen) [inline tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100]: Does this line do anything?

Comment (binli2337) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100]: They are not used. I can remove them. Thanks.

**Review by DeniseWorthen — APPROVED** [review]
I looked at your run directories on orion and all the output files have the right timestamp so I believe these are all working correctly. I do think we need to add instructions for running the test on the wiki, maybe under a new heading '3. Running the 35-d test suite' or something.

**Review by junwang-noaa — APPROVED** [review]
The code change look good to me. I may suggest in the future to set up a default configuration file for all those 35 days  test cases (tests/tests/*_35d_*) since they should be run with same configuration, it will be easy to maintain if the configurations are in one file, rather than 8 files.

File: tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100 (hunk 1 above)
Comment (MinsukJi-NOAA) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100]: When I tried to run the warm 35d tests, I get `CNTL_DIR: unbound variable` error. Looks like these lines, even though they may not be used, need to be in place.

Comment (DeniseWorthen) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100]: When I was doing the cmeps commit, I had added some empty variables to default_vars (which I don't think I ended up using):

export MEDCOMP=''
export RESTART_PREFIX=''
export RESTART_SUFFIX=''
}

I think you could do the same thing here? Add CNTL_DIR as an empty string in default_vars.

Comment (MinsukJi-NOAA) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100]: Yes. If that does not work, it will have to be fixed from the script side.

**Pull request merged**